docs/en/registry-catalogue.rst

```diff
@@ -0,0 +1,338 @@
+       - **Digital Credential purposes**: Information related to the allowed purposes for which the Digital Credential can be used. Each Digital Credential type can be used for multiple purposes.
+   * - Claims and Taxonomy References
+     - Content and classification information:
+       
+       - **List of displayed claims**: Specific Digital Credential content displayed to the User.
+       - **Structured taxonomy references**: Classification systems and controlled vocabularies used.
+
+
+The Trust Anchor MUST publish and keep up to date all the information at the Digital Credential Catalogue `.well-known` endpoint ensuring data reliability, authenticity and integrity. In particular, the Digital Credential Catalogue, claims and taxonomy MUST be available through the ``.well-known/credential-catalogue`` endpoint.
+
+Digital Credentials Categories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Digital Credentials recognized within the IT-Wallet ecosystem are hierarchically classified and standardized according to the following main domains and categories. Additional categories MAY be added as the IT-Wallet ecosystem grows.
+
+.. list-table:: Digital Credential Domains and Categories
+   :header-rows: 1
+   :widths: 20 30 50
+
+   * - **Domain**
+     - **Category**
+     - **Description**
+   * - *IDENTITY*
+     - * PERSON_IDENTIFICATION
+       * ELECTRONIC_RESIDENCY
+     - Credentials that establish or verify the identity of a person, including physical and digital identity documents legally recognized by national laws.
+   * - *AUTHORIZATION*
+     - * DRIVING_LICENSE
+       * PROFESSIONAL_LICENSE
+       * TRAVEL_DOCUMENT
+       * ACCESS_PERMIT
+     - Credentials that grant specific permissions, rights or authorizations to perform certain activities or access restricted areas.
+   * - *EDUCATION*
+     - * ACADEMIC_DEGREE
+       * CERTIFICATE
+       * TRAINING_RECOGNITION
+     - Credentials related to educational achievements, qualifications, and professional training recognition.
+   * - *HEALTH*
+     - * INSURANCE_CARD
+       * DISABILITY_CARD
+       * MEDICAL_PRESCRIPTION
+     - Credentials related to healthcare access, medical history, insurance coverage, and health-related documents.
+   * - *FINANCIAL*
+     - * INCOME_CERTIFICATE
+       * TAX_STATEMENT
+       * FAMILY_ECONOMIC_STATUS
+     - Credentials that attest to financial status, income levels, taxation, or economic situation of individuals or families.
+   * - *MEMBERSHIP*
+     - * ASSOCIATION
+       * LOYALTY_PROGRAM
+       * CLUB_MEMBERSHIP
+     - Credentials that confirm affiliation with organizations, participation in programs, or membership status.
+   * - *ATTESTATION*
+     - * PUBLIC_STATEMENT
+       * CIVIL_STATUS
+       * CERTIFICATION
+     - Credentials that provide official statements, confirmations of status, or certifications issued by authorities.
+
+
+Digital Credentials Catalogue Structure
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I would like to have the definition of an http endpoint with http request content type set to jose or json

is the request cames with content type jose, the response must be JWS, otherwise plain json

Reply:
see my comment below

Reply:
The content type is already set to jose, thus the response is JWS. Maybe I'm missing something, can you clarify your comment?

Reply:
developers uses to navigate with their user agent to inspect the responses and there might be situation where having a plain text would facilitate dev exp

we can move this into a separate issue for next release

Reply:
https://github.com/italia/eid-wallet-it-docs/issues/641